### ISSUE TITLE
LDAP not working when @name is used

### ISSUE THREAD

**Issue by Fabian1976** [djjudas21/puppet-freeradius#118]
https://github.com/djjudas21/puppet-freeradius/blob/39cddccc46f7a8da5812db30d5029e275b62db14/templates/ldap.erb#L8

When I configure a LDAP server using create_resource from hiera:
```
freeradius::module::ldap:
  'name_of_ldap':
    server: '192.168.0.1'
    ....
```
Then an entry gets created in the folder mods-enabled named 'name_of_ldap' with this content:
```
ldap name_of_ldap {
  server = '192.168.0.1
  ....
}
```
Which seems correct, but the FreeRadius server won't start and this error appears in the log:
```
Error: /etc/raddb/sites-enabled/name_of_ldap[75]: Unknown attribute 'LDAP-Group'
```

BUT when I remove name_of_ldap from the mod, so then it looks like this:
```
ldap {
  server = '192.168.0.1
  ....
}
```
It works fine. Am I missing something? Or is the code not correct.

**Comment by djjudas21** [comment]
Hi, thanks for submitting this bug report. What you've done looks correct. At a guess, I think it isn't loading `rlm_ldap`. Does it work without the `@name` param? What if you load a default base LDAP without `@name` and then continue to add your named LDAP module too?

Unfortunately I don't use this module any more, and I didn't write the LDAP code so I don't think I can really be of much help. If you can figure out the problem, I will gladly consider a pull request though.

**Comment by Fabian1976** [comment]
Hi.
Without the ```@name``` parameter, it does work.

I've also tried enabling the default LDAP module:
```
freeradius::module:
  'ldap':
    preserve: true
```
but then it still doesn't work properly. It can load the LDAP module, but it still won't start because of connection issues.

I've created a fork and removed the ```@name``` from the template and used that module to test it, and then all works fine.

I will create a pull request with that change.
